test/util.spec.js

```diff
@@ -2405,4 +2407,49 @@ describe('UTIL', function() {
+            nativeAdString= '{"native":{"assets":[{"id":1,"required":0,"title":{"text":"Lexus - Luxury vehicles company"}},{"id":2,"img":{"h":150,"url":"https://stagingnyc.pubmatic.com:8443//sdk/lexus_logo.png","w":150},"required":0},{"id":3,"img":{"h":428,"url":"https://stagingnyc.pubmatic.com:8443//sdk/28f48244cafa0363b03899f267453fe7%20copy.png","w":214},"required":0},{"data":{"value":"Goto PubMatic"},"id":4,"required":0},{"data":{"value":"Lexus - Luxury vehicles company"},"id":5,"required":0},{"data":{"value":"4"},"id":6,"required":0}],"imptrackers":["http://phtrack.pubmatic.com/?ts=1496043362&r=84137f17-eefd-4f06-8380-09138dc616e6&i=c35b1240-a0b3-4708-afca-54be95283c61&a=130917&t=9756&au=10002949&p=&c=10014299&o=10002476&wl=10009731&ty=1"],"link":{"clicktrackers":["http://ct.pubmatic.com/track?ts=1496043362&r=84137f17-eefd-4f06-8380-09138dc616e6&i=c35b1240-a0b3-4708-afca-54be95283c61&a=130917&t=9756&au=10002949&p=&c=10014299&o=10002476&wl=10009731&ty=3&url="],"url":"http://www.lexus.com/"},"ver":1}}';
+            bannerAdString = '<img src="http://ads.pubmatic.com/AdTag/728x90.png"></img><div style="position:absolute;left:0px;top:0px;visibility:hidden;"><img src="http://sin1-ib.adnxs.com/it?e=wqT_3QKqB6CqAwAAAwDWAAUBCIKHsdoFELzlh5jb&s=dfa92ae59f49b6052953a52fcc0152434618139a&referrer=http%253A%252F%252Febay.com%252Finte%252Fappnexus.html%253Fpwtvc%253D1%2526pwtv%253D2%2526profileid%253D2514"></div>'
+            done();
+        });
+
+        afterEach(function(done){
+            adFormat= videoAdString=nativeAdString=bannerAdString= undefined;
+            done();
+        });
+
+        it('should be a function',function(done){
+            UTIL.getAdFormatFromBidAd.should.be.a('function');
+            done();
+        });
+
+        // it('should return video if ad html consists of video string ',function(done){
+        //     var expectedResult = CONSTANTS.FORMAT_VALUES.VIDEO;
+        //     var oParser = new DOMParser();
+        //     var oDOM = oParser.parseFromString(videoAdString, "text/xml");
+        //     var result =  UTIL.getAdFormatFromBidAd(oDOM);
+        //     result.should.deep.equal(expectedResult);
+        //     done();
+        // });
+
+        it('should return native if ad html consists of video string ',function(done){
+            var expectedResult = CONSTANTS.FORMAT_VALUES.NATIVE
+            var result =  UTIL.getAdFormatFromBidAd(nativeAdString);
+            result.should.deep.equal(expectedResult);
+            done();
+        });
+
+        it('should return banner if ad html consists of video string ',function(done){
```

Comment:
"should return banner if ad html consists of video string" ?
it should have been "should return banner if ad html consists of banner string"

Reply:
Already fixed on local and forgot to push it.
Fixed Now